server/routes/team.router.js

```diff
@@ -39,6 +45,9 @@ router.get('/leagueTeam/:id', rejectUnauthenticated, (req, res) => {
   });
 });
 
+/**
+ * GET access codes?
+ */
```

Comment:
We need to ✂️  cut this endpoint. It's not in use anymore, (right, @AlvinWong13 ?)